src/client/pages/ConnectPayment/sections/ConnectPayment.tsx

```diff
@@ -106,10 +108,17 @@ const ConnectPaymentImage = styled('img')({
 export const ConnectPaymentPage: React.FC = () => {
   const textKeys = useTextKeys()
   const market = useMarket()
+  const variation = useVariation()
 
   const onSuccess = () => {
-    const message = JSON.stringify({ event: 'PaymentConnected' })
-    window.frames.parent.postMessage(message, '*')
+    if (Variation.AVY === variation) {
```

Comment:
Are these kind of "yoda conditions" intentional? (Yes, that's the name, see [wikipedia](https://en.wikipedia.org/wiki/Yoda_conditions) if it sounds unfamiliar 😄) Personally I'm not sure i'm a fan, especially in this flavor of JS since everything is kind of constant anyways.

Reply:
change this, I will!